csharp/generate/config.json

```diff
@@ -1,9 +1,9 @@
 {
   "packageName": "Lusid.Sdk",
   "sourceFolder": ".",
-  "targetFramework": "v5.0",
+  "targetFramework": "net5.0",
```

Comment:
We have a dependency on the C# SDK for integration testing. Most of our applications currently use .net core 3.1, I would imagine a lot of clients are also on 3.1, will this be usable by those applications if it is .net 5.0?

Further we currently target `netstandard2.0` combined with the flag change of `-g csharp-netcore` in `generate.sh` will users of .Net Framework be able to use the SDK?